task-JDP201202-2

wezik opened the pull request:
Wstępna implementacja GroupController
Utworzenie GroupDto
@danieloginski

wezik committed:
GroupController - Wstępna implementacja endpointów

Wstępna implementacja GroupController
Utworzenie GroupDto

danieloginski commented on src/main/java/com/kodilla/ecommercee/dto/GroupDto.java:
```diff
@@ -1,4 +1,11 @@
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
```
zamiast tego, proponuję użyć `@Value` - https://projectlombok.org/features/Value

danieloginski reviewed (changes requested):
@wezik

danieloginski commented on src/main/java/com/kodilla/ecommercee/controller/GroupController.java:
```diff
@@ -1,4 +1,38 @@
+        return Arrays.asList(groups);
+    }
+
+    @GetMapping(value = "getGroup")
```
`"getGroup/{id}"`

wezik committed:
Korekty GroupController i GroupDto

GroupDto:
- Zamiana adnotacji @Getter i @AllArgsConstructor na jedną @Value

GroupController:
- zmieniony sposób przekazywania danych w getGroup

wezik commented:
Wprowadzone zmiany
@danieloginski
